tests/scripts/remote_monitoring_tests/helpers/utils.py

```diff
@@ -68,7 +68,9 @@
 CRITICAL_MEMORY_LIMIT_NOT_SET_CODE = "524002"
 
 INFO_COST_RECOMMENDATIONS_AVAILABLE_CODE = "112101"
+INFO_PERFORMANCE_RECOMMENDATIONS_AVAILABLE_CODE = "112102"
```

Comment:
I don't see this being used anywhere. Please include tests for validating the notification codes.